### DISCUSSION TITLE
Keycloak 19.0.3 using Quarkus loosing connection to DB (agroal)

### DISCUSSION THREAD

**Discussion by mortenjoenby** [quarkusio/quarkus#31813] [Community]
Hi all.
I created this Keycloak issue last week, so cross-posting a link here with the hope that someone might have seen this issue where DB connection is not re-established.
https://github.com/keycloak/keycloak/issues/17485

Thanks,
Morten

**Comment by sberyozkin** [comment]
CC @gsmet Guillaume, please cc to those who may know about Agroal internals
